### PR TITLE
DOCS-1113 Create ai accelerator ga documentation section

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

First draft commits WIP

### CODE REVIEW

File: advocacy_docs/edb-postgres-ai/overview/guide-and-getting-started.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ You'll want to look at the [EDB Postgres® AI Analytics](/edb-postgres-ai/analyt
 
 ## Are you looking at running machine learning models on your Postgres data?
 
-You'll want to look at the [EDB Postgres® AI Machine Learning](/edb-postgres-ai/ai-ml) documentation, which covers the technical preview of the aidb extension.
+You'll want to look at the [EDB Postgres® AI Machine Learning](/edb-postgres-ai/ai-accelerator/) documentation, which covers the technical preview of the aidb extension.
```
Comment (gvasquezvargas) [inline advocacy_docs/edb-postgres-ai/overview/guide-and-getting-started.mdx:13]:
```suggestion
You'll want to look at the [EDB Postgres® AI - AI Accelerator](/edb-postgres-ai/ai-accelerator/) documentation, which covers the technical preview of the aidb extension.
```

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx [hunk 2]
```diff
@@ -47,8 +47,9 @@ Filtering out the data noise and revealing insights and value, Lakehouse analyti
 -   **[Lakehouse nodes](/edb-postgres-ai/analytics/concepts/)**
     -   At the heart of Analytics is custom-built object storage for your data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring cold data in, ready for analysis. 
 
-## [EDB Postgres AI AI/ML](/edb-postgres-ai/ai-ml)
-* Postgres has proven its capability as a flexible data environment. With vector data as the core of generative AI, it's already infused into EDB Postgres AI, providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres aidb extension.
+## [EDB Postgres AI AI/ML](/edb-postgres-ai/ai-accelerator/)
+
+-   Postgres has proven its capability as a flexible data environment. With vector data as the core of generative AI, it's already infused into EDB Postgres AI, providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres aidb extension.
```
Comment (gvasquezvargas) [inline advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:52]: Not sure if to call it first release, or GA
```suggestion
-   Postgres has proven its capability as a flexible data environment. With vector data as the core of generative AI, it's already infused into EDB Postgres AI, providing a platform for a range of practical and effective AI/ML solutions. The first release of this capability is available for the Postgres aidb extension.
```

File: advocacy_docs/edb-postgres-ai/ai-accelerator/pipelines/installing/packages.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+### Installing the packages
+
+Once the server is running, there will be a number of packages that need to be installed. The packages are available in the EDB repository, and can be installed using the appropriate package manager for your operating system. You will need to use either:
+
+- `apt` for Debian/Ubuntu based systems with .deb packages
+- `dnf` for Red Hat based systems with .rpm packages
+
+Depending on your operating system. The packages are:
+
+#### For Community PostgreSQL
```
[the remark this answers is not in the document]
Comment (djw-m) [reply advocacy_docs/edb-postgres-ai/ai-accelerator/pipelines/installing/packages.mdx:35]: Not keen on making everything fold, and ideally we want to get a folding component to make it more reusable.

File: advocacy_docs/edb-postgres-ai/ai-accelerator/pipelines/models/supported-models/openai-embeddings.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+* Any text embedding model that is supported by OpenAI. This includes `text-embedding-3-small`, `text-embedding-3-large`, and `text-embedding-ada-002`.
+* Defaults to `text-embedding-3-small`.
+
+## Registering the default model
+
+```sql
+SELECT aidb.register_model('my_openai_embeddings', 
+                            'openai_embeddings',
+                            credentials=>'{"api_key": "sk-abc123xyz456def789ghi012jkl345mn"'::JSONB);
```
[the remark this answers is not in the document]
Comment (djw-m) [reply advocacy_docs/edb-postgres-ai/ai-accelerator/pipelines/models/supported-models/openai-embeddings.mdx:30]: Already random.

**Pull request merged**